### PR TITLE
Carinfo firebase service

### PR DESCRIPTION
**Pull request by luisgrases**

### CODE REVIEW

File: plugins/android.json [hunk 1]
```diff
@@ -4,7 +4,556 @@
         "uninstalled": []
     },
```
Comment (joserv2812) [inline plugins/android.json]: add to .gitignore

File: tests/specs/carInfo/carInfo.services.spec.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    beforeEach(function() {
+      module(function($provide) {
+        $provide.service('LoginFirebaseService', function() {
+          return {'username': 'foo'};
+        });
+      });
+    });
+
+    beforeEach(inject(function(_$firebaseObject_, $q, _LoginFirebaseService_, _CarInfoFirebaseService_) {
```
Comment (joserv2812) [inline tests/specs/carInfo/carInfo.services.spec.js]: why are you injecting $q?

Comment (joserv2812) [inline tests/specs/carInfo/carInfo.services.spec.js]: and _$firebaseObject_

File: www/styles/login.css [hunk 3]
```diff
@@ -28,52 +28,46 @@
...
 }
 
-@media only screen 
-and (min-width :850px) {
-  .budget-logo{
-    margin-top: 10%;
+@media only screen
+and (min-width: 850px) {
+  .budget-logo {
+      margin-top: 10%;
```
Comment (joserv2812) [inline www/styles/login.css]: indentation

File: www/styles/login.css [hunk 4]
```diff
@@ -28,52 +28,46 @@
...
-  .user-information-box{
-    font-size: 2em;
+  .user-information-box {
+      font-size: 2em;
   }
 
-  a{
-    font-size: 2em;
+  a {
+      font-size: 2em;
```
Comment (joserv2812) [inline www/styles/login.css]: indentation

File: www/styles/login.css [hunk 5]
```diff
@@ -28,52 +28,46 @@
...
 }
 
-@media only screen 
-and (min-width :650px) {
-  .budget-logo{
-    margin-top: 50%;
+@media only screen
+and (min-width: 650px) {
+  .budget-logo {
+      margin-top: 50%;
```
Comment (joserv2812) [inline www/styles/login.css]: indentation

File: www/styles/carView.css [hunk 6]
```diff
@@ -118,16 +130,16 @@ canvas[resize] {
...
   }
 }
 
-@media only screen 
+@media only screen
 and (min-device-height: 1500px) {
-  canvas-wrapper{
-    margin-top: 5em;
+  canvas-wrapper {
+      margin-top: 5em;
```
Comment (joserv2812) [inline www/styles/carView.css]: indentation

File: www/styles/carView.css [hunk 7]
```diff
@@ -118,16 +130,16 @@ canvas[resize] {
   height: 8em;
 }
 
-@media only screen 
+@media only screen
 and (max-device-width: 800px) {
   canvas[resize] {
-    height: 68em;
+      height: 68em;
```
Comment (joserv2812) [inline www/styles/carView.css]: indentation

File: www/styles/carInfo.css [hunk 8]
```diff
@@ -22,29 +22,29 @@
...
-    height: 5em;
+  .car-info-tdata {
+      width: 4em;
+      height: 5em;
   }
 
-  .content-wrapper{
-    margin-top: 5em !important;
+  .content-wrapper {
+      margin-top: 5em !important;
```
Comment (joserv2812) [inline www/styles/carInfo.css]: indentation

File: www/styles/carInfo.css [hunk 9]
```diff
@@ -22,29 +22,29 @@
...
 
-@media only screen 
+@media only screen
 and (min-width: 650px) {
 
-  .car-info-tdata{
-    width: 4em;
-    height: 5em;
+  .car-info-tdata {
+      width: 4em;
```
Comment (joserv2812) [inline www/styles/carInfo.css]: indentation

File: www/app/login/login.controller.js [hunk 10]
```diff
@@ -36,10 +28,13 @@
           try{
             if(username === snapshot.val().username && password === snapshot.val().password){
               vm.authSuccess();
+              LoginFirebaseService.username = username;
             }else{
+              
               vm.authError();
             }
           }catch(err){
+            console.log(err);
```
Comment (joserv2812) [inline www/app/login/login.controller.js]: don't use console.log

File: www/app/carInfo/carInfo.services.js [hunk 11]
```diff
@@ -3,18 +3,71 @@
```
Comment (joserv2812) [inline www/app/carInfo/carInfo.services.js:14]: use carInfo.service.js

File: www/app/carInfo/carInfo.services.js [hunk 12]
```diff
@@ -3,18 +3,71 @@
...
 
-    function carInfoService($firebaseObject, firebase_url){
-      return {
-        getVehicle : getVehicle
-      };
-      
-      function getVehicle(code){
-          var firebaseRef = new Firebase(firebase_url + "/vehicles/" + code);
-          return $firebaseObject(firebaseRef);
+      var _model = {
```
Comment (joserv2812) [inline www/app/carInfo/carInfo.services.js]: use more significant names

File: www/app/carView/carView.html [hunk 13]
```diff
@@ -21,7 +21,7 @@
       </ion-item>
     </ion-list>
   </ion-scroll>
-  <div class="arrow-wrapper-carView">
-    <img ng-click="goToExteriorParts()" src="assets/images/arrow.png" class="arrow carview-arrow arrow-carView-style">
-  </div>
+    <div class="btn-continue-wrraper">
+        <button class="button button-positive btn-continue" ng-click="goToExteriorParts()"> Continuar </button>
```
Comment (joserv2812) [inline www/app/carView/carView.html]: indentation

File: www/app/carInfo/carInfo.services.js [hunk 14]
```diff
@@ -3,18 +3,71 @@
...
+        rootRef : new Firebase('https://budget-test.firebaseio.com'),
+        getCarInfo: getCarInfo,
+        fillNewRevisionData: fillNewRevisionData,
+        pushNewRevision: pushNewRevision,
+        carInfo: null,
+        currentCarId: ScannerService.getCode(),
+        currentRevisionId: '56456165',
```
Comment (johnnyxl26) [inline www/app/carInfo/carInfo.services.js]: Use autogenerated id from firebase.

**Comment by joserv2812** [comment]
fix conflicts

**Pull request merged**